### DISCUSSION TITLE
My sim doesn’t want to load

### DISCUSSION THREAD

**Discussion by Paolo131994** [flybywiresim/aircraft#7191] [Help]
Hi Guys, 

I’ve made the experimental update today and when I try to launch a flight the loading page is appearing, the loading bar finish to load but I’m waiting 10 min and nothing is happening. The flight is still in permanent load. Any idea or suggestions?

best regards 

Paolo

**Comment by andrew1244** [comment]
Ormai ho rinunciato a far volare l'A320,.  Dopo innumerevoli tentativi e attese con  cambio continuo di mod, stable , development e experimental ho gettato la spugna, Sono passato al A330 900 che usa la stessa tecnologia FBW.
